tests/budgeting/rules/test_rules_view_vote_count.py

```diff
@@ -0,0 +1,335 @@
+    (
+        project,
+        group_member_in_org,
+        group_member_in_pro,
+        group_member_out,
+    ) = setup_group_members(project, group_factory, user_factory)
+
+    assert project.is_public
+    with freeze_phase(voting_phase):
+        assert not rules.has_perm(perm_name, anonymous, module)
+        assert not rules.has_perm(perm_name, user, module)
+        assert not rules.has_perm(perm_name, group_member_out, module)
+        assert not rules.has_perm(perm_name, group_member_in_org, module)
+        assert not rules.has_perm(perm_name, group_member_in_pro, module)
+        assert not rules.has_perm(perm_name, moderator, module)
+        assert not rules.has_perm(perm_name, initiator, module)
+        assert rules.has_perm(perm_name, admin, module)
+
+
+@pytest.mark.django_db
+def test_voting_phase_over(
+    phase_factory, proposal_factory, user, admin, user_factory, group_factory
+):
+    voting_phase, module, project, item = setup_phase(
+        phase_factory,
+        proposal_factory,
+        phases.VotingPhase,
+    )
+    anonymous, moderator, initiator = setup_users(project)
+    (
+        project,
+        group_member_in_org,
+        group_member_in_pro,
+        group_member_out,
+    ) = setup_group_members(project, group_factory, user_factory)
+
+    with freeze_post_phase(voting_phase):
+        assert rules.has_perm(perm_name, anonymous, module)
+        assert rules.has_perm(perm_name, user, module)
+        assert rules.has_perm(perm_name, group_member_out, module)
+        assert rules.has_perm(perm_name, group_member_in_org, module)
+        assert rules.has_perm(perm_name, group_member_in_pro, module)
+        assert rules.has_perm(perm_name, moderator, module)
+        assert rules.has_perm(perm_name, initiator, module)
+        assert rules.has_perm(perm_name, admin, module)
+
+
+@pytest.mark.django_db
+def test_voting_phase_over_project_private(
+    phase_factory, proposal_factory, user, admin, user_factory, group_factory
+):
+    voting_phase, module, project, item = setup_phase(
+        phase_factory,
+        proposal_factory,
+        phases.VotingPhase,
+        module__project__access=Access.PRIVATE,
+    )
+    anonymous, moderator, initiator = setup_users(project)
+    (
+        project,
+        group_member_in_org,
+        group_member_in_pro,
+        group_member_out,
+    ) = setup_group_members(project, group_factory, user_factory)
+
+    participant = user_factory()
+    project.participants.add(participant)
+
+    assert project.access == Access.PRIVATE
+    with freeze_post_phase(voting_phase):
+        assert not rules.has_perm(perm_name, anonymous, module)
+        assert not rules.has_perm(perm_name, user, module)
+        assert not rules.has_perm(perm_name, group_member_out, module)
+        assert not rules.has_perm(perm_name, group_member_in_org, module)
+        assert not rules.has_perm(perm_name, group_member_in_pro, module)
```

Comment:
Not sure about this one.. this is false, because we replaced the `is_allowed_moderate_project` by `is_superuser` here. Same for initiators below.. I guess both of them should see the vote count in a private project though, because they are allowed to see the private project, right? @fuzzylogic2000 But not sure how to change that (in a consistent way).. :exploding_head:

Reply:
Ohhhhh, very right. Because we use that for determining if we want to show the moderation stuff, right? Ah. Should we look at that together?

Reply:
I added that comment to an issue we have about that: https://github.com/liqd/a4-meinberlin/issues/4717
It's also in the prod release milestone, so we sill have to come back to it. :) So, with that, we are good to merge! But I am not sure, I am meant to? I wasn't assigned a s rewiewer, but found this through your comments. So, feel free to merge if it is ready!

Reply:
@fuzzylogic2000 I wanted to add some more tests, so lets wait for that? Or do you want to have it in the release?

Reply:
Cool! No, waiting is fine!